### ISSUE TITLE
Empty spans rendered as invalid json

### ISSUE THREAD

**Issue by gavares** [zio/zio-logging#783]
zio-logging version: 2.1.14

Used with:
- zio-logging-slf4j2-bridge: 2.1.14

Log configuration is read from HOCON with format string:
```
logger.formatformat =     "%label{timestamp}{%timestamp} %label{level}{%level} %label{fiberId}{%fiberId} %label{message}{%message} %label{cause}{%cause} %label{loggerName}{%name} %label{spans}{%spans} %kvs"
```

Any calls to log methods in scopes which do not have defined spans result in invalid json. For example, a logDebug message printed from a webserver filter:

```
logDebug("a log message")
```

Results in:
```
{
  "timestamp":"2023-11-04T13:47:10.732971635Z",
  "level":"DEBUG",
  "fiberId":"zio-fiber-409138",
  "message":"a log message",
  "loggerName":"sttp.tapir.server.ziohttp.ZioHttpServerOptions",
  "spans":
}
```
_newlines added for readability_

You can see in the last line that logging with no defined spans prints a key, there is a call to `LogAppender.closeKeyOpenValue()` and then nothing is printed. 

The right behavior here is likely one of:

- omit `spans` key if `List[LogSpan[` is empty
- print `null` if `List[LogSpan]` is empty
- print `{}` if `List[LogSpan]` is empty

**Comment by justcoon** [comment]
@gavares thank you very much for report, 
look like there is general issue in case where json attribute value is empty, and does not matter if it is span or something else,  

I created fix for that #784

**Comment by gavares** [comment]
Wow, thanks for the quick response @justcoon
